src/Microsoft.AspNetCore.CertificateGeneration.Task/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.CertificateGeneration.Task
+{
+    public class Program
+    {
+        public static int Main(string[] args)
+        {
+            var app = new CommandLineApplication();
+            app.HelpOption("-h|--help");
+
+            app.Command("ssl", c =>
```

Comment:
How about "self-sign" or "self-signed" instead of "ssl"? "ssl" is too generic/vague.